### DISCUSSION TITLE
Does statusline suppose to look like this?

### DISCUSSION THREAD

**Discussion by karambaq** [olimorris/onedarkpro.nvim#211] [Q&A]
Hey, I'm using this theme with astronvim and I'm not sure is it a bug or no but statusline and bufferline have exact same background as the editor:

<img width="1582" alt="image" src="https://github.com/olimorris/onedarkpro.nvim/assets/14233969/fe4124ec-9eb6-479c-aa81-98b2991963e8">

**Comment by olimorris** [comment]
I think that's one for the Astronvim community. If you're using Lualine then the background should be different.
